conf/routes

```diff
@@ -8,4 +8,5 @@ GET         /install/                      @controllers.github.InstallHooks.inst
 POST        /install/                      @controllers.github.InstallHooks.installPost
 GET         /static/*name                  @controllers.StaticDev.get(name: String)
 GET         /version/                      @controllers.Version.version
+GET         /installBitbucket/             @controllers.github.InstallHooks.installGetBitbucket
```

Comment:
Typically in REST preferable to have `/install-bitbucket/` and not camel case.

I'll take note in making this explicit in future.